stdlib/ftplib.pyi

```diff
@@ -31,7 +31,7 @@ class FTP:
     sock: socket | None
     welcome: str | None
     passiveserver: int
-    timeout: int
+    timeout: float | None
```

Comment:
Agree that this could be `None` -- the implementation specifically checks to see whether or not it's `None`: https://github.com/python/cpython/blob/46245b0d831b9f5b15b4a0483c785ea71bffef12/Lib/ftplib.py#L153-L154

Reply:
The `timeout` value is also eventually passed to `socket.create_connection` here: https://github.com/python/cpython/blob/46245b0d831b9f5b15b4a0483c785ea71bffef12/Lib/ftplib.py#L158-L159

And our annotations for `create_connection` state that `None` is accepted:

https://github.com/python/typeshed/blob/41245698e07a9959a4643122ddc814dff1ad3482/stdlib/socket.pyi#L803-L815